GameDefinition/Content/ItemKindOrgan.hs

```diff
@@ -186,7 +186,7 @@ venomTooth = fist
   , idamage  = toDmg $ 2 `d` 1
   , iaspects = [Timeout $ 5 + 1 `d` 3]
   , ieffects = [Recharging (toOrganGameTurn "slowed" (3 + 1 `d` 3))]
-  , idesc    = ""
+  , idesc    = "A hollow fang, filled with venom."
```

Comment:
Again, I'd somehow differentiate the slowing/paralysing with pain/distracting venom from the poisoning one below.